### PR TITLE
Fix up unitdef post styling

### PR DESCRIPTION
**Pull request by Arch-Shaman**
Because I may want to eventually touch this for #4149.

### CODE REVIEW

File: gamedata/unitdefs_post.lua [hunk 1]
```diff
@@ -786,9 +799,9 @@ for name, ud in pairs(UnitDefs) do
 	
 	if VISUALIZE_SELECTION_VOLUME then
 		if ud.selectionvolumescales then
-			ud.collisionvolumeoffsets = ud.selectionvolumeoffsets
-			ud.collisionvolumescales  = ud.selectionvolumescales
-			ud.collisionvolumetype    = ud.selectionvolumetype
+			ud.collisionvolumeoffsets	= ud.selectionvolumeoffsets
+			ud.collisionvolumescales	= ud.selectionvolumescales
```
Comment (sprunk) [inline gamedata/unitdefs_post.lua:803]: no tabs for alignment

File: gamedata/unitdefs_post.lua [hunk 2]
```diff
@@ -923,8 +936,8 @@ if Utilities.IsCurrentVersionNewerThan(104, 600) then
 	end
 else
 	--[[ old engines handle transporting rules entirely on their own,
-	     but mark units anyway so that other code doesn't need to
-	     replicate these checks ]]
+		 but mark units anyway so that other code doesn't need to
```
Comment (sprunk) [inline gamedata/unitdefs_post.lua:939]: no tabs for alignment

File: gamedata/unitdefs_post.lua [hunk 3]
```diff
@@ -923,15 +936,15 @@ if Utilities.IsCurrentVersionNewerThan(104, 600) then
 	end
 else
 	--[[ old engines handle transporting rules entirely on their own,
-	     but mark units anyway so that other code doesn't need to
-	     replicate these checks ]]
+             but mark units anyway so that other code doesn't need to
```
Comment (sprunk) [inline gamedata/unitdefs_post.lua:939]: no spaces for intentation! :P
(should be 1 tab and then 5 spaces for text alignment, i.e. kept unchanged)

Comment (Arch-Shaman) [reply gamedata/unitdefs_post.lua:939]: haha okay. one extreme to the other.

**Pull request merged**